src/main/java/uk/gov/ons/ctp/integration/rhsvc/representation/WebformDTO.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.ons.ctp.integration.rhsvc.representation;
+
+import java.io.Serializable;
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.ons.ctp.common.domain.Region;
+
+/** This object holds details from a webform. */
+@Data
+@NoArgsConstructor
+@SuppressWarnings("serial")
+public class WebformDTO implements Serializable {
```

Comment:
remove SuppressWarnings and implementing Serializable. Not needed.

Reply:
Removed